platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -92,7 +93,7 @@
    * @throws MigrationException to stop the import operation
    */
   public default void doVersionUpgradeImport(
-      ImportMigrationContext context, String migratableVersion) {
+      ImportMigrationContext context, Properties migrationProperties, String migratableVersion) {
```

Comment:
❓Can we avoid passing in all the `Properties` and only provide the one/ones the version upgrade code will need instead? Passing in all of them creates implicit coupling and becomes a dumping ground for all sorts of key/value pairs in the future that's difficult to track and maintain.

Reply:
These Properties will no longer be passed in in the next commit, per our offline conversation.